src/main/java/org/simplify4u/plugins/PGPVerifyMojo.java

```diff
@@ -132,6 +132,12 @@
     @Parameter(property = "pgpverify.failNoSignature", defaultValue = "false")
     private boolean failNoSignature;
 
+    /**
+     * Fail the build if any artifact without key is not present in the keys list.
+     */
+    @Parameter(property = "pgpverify.strictNoSignature", defaultValue = "false")
+    private boolean strictNoSignature;
```

Comment:
please add @ since 1.5.0 javadoc annotation

please also add information that pgp key can by empty string near line 214, in documentation for keysMapLocation